Alm/SuperbClass.cs

```diff
@@ -1,10 +1,22 @@
-﻿namespace Alm
+﻿using System;
+
+namespace Alm
 {
-    public class SuperbClass
-    {
+	public class SuperbClass
+	{
 		public int TheAnswer() => 42;
 
 		public string TheQuestion() => "the Ultimate Question of Life, The Universe, and Everything";
+	}
 
+	public class AnotherClass
+	{
+		public void IThrowExceptionsBaaadly(int answer)
+		{
+			if (answer != 42)
+			{
+				throw new Exception("Don't throw this please");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') 'System.Exception' should not be thrown by user code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS112)